### DISCUSSION TITLE
为什么截图时会显示键盘状态 numlock

### DISCUSSION THREAD

**Discussion by chijialone** [xushengfeng/eSearch#177] [Q&A] [answered]
![image](https://github.com/xushengfeng/eSearch/assets/93021384/a8f3b233-a1ac-4a37-8be3-7e502c55d664)

截图时会显示键盘状态 numlock，如图，有什么办法不显示吗

**Comment by xushengfeng** [comment, marked as answer]
见#124
测试版修复了 https://github.com/xushengfeng/eSearch/releases

**Reply by chijialone** [reply]
谢谢
